### PR TITLE
Added summaries for Speusippus and Xenocrates of Chalcedon

### PR DESCRIPTION
**Pull request by ashelkovnykov**

### CODE REVIEW

File: _posts/philosophers/2020-06-30-seusippus.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags: [platonist, greek]
+---
+Born some time in 408 BC.
+Died either 339 BC or 338 BC (age ~70).
+
+Speusippus was a Greek philosopher. He was a nephew of Plato, and his successor as Scholarch of The Old Academy. Speusippus ran the Academy for eight years after Plato's death, until a stroke forced him to resign.
+
+The ideas of Speusippus were similar to those of Plato, but different in several major ways. Speusippus rejected Plato's Doctrine of Forms. He asserted that The One and The Good were not the same. Instead, there are multiple substances (The Many) which stem from The One and which are the basis for the principles of the universe. The Good is the ultimate result, not source, of these principles. He identified the gods with the principles, acting as physical forces, which led to a charge of atheism brought against him.
+
+Speusippus believed in the immortality of the soul. However, he asserted the the whole human soul was immortal, not only the rational part. He also believed that perception could be transformed into true knowledge through determination of how perceptions participate in rational truth.
```
Comment (joshvfleming) [inline _posts/philosophers/2020-06-30-seusippus.md:15]: Duplicated "the":
```
... he asserted the the whole human soul ...
                    ^^^
```

Comment (joshvfleming) [inline _posts/philosophers/2020-06-30-seusippus.md:15]: Duplicated "the":
```
... he asserted the the whole human soul ...
                    ^^^
```

Comment (ashelkovnykov) [reply _posts/philosophers/2020-06-30-seusippus.md]: Fixed, thank you

**Comment by ashelkovnykov** [comment]
@joshvfleming @philozophy 
- Fixed typos
- Updated date

**Pull request merged**